presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1172,6 +1180,20 @@ std::vector<std::string> PrestoServer::registerConnectors(
     const fs::path& configDirectoryPath) {
   static const std::string kPropertiesExtension = ".properties";
 
+  const auto numConnectorCpuThreads = std::max<size_t>(
```

Comment:
you might want to track the connector cpu executor stats in followup. Thanks!

Reply:
Will check with you about it offline